src/category_theory/limits/shapes/constructions/equalizers.lean

```diff
@@ -0,0 +1,79 @@
+/-
+Copyright (c) 2020 Bhavik Mehta. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Bhavik Mehta
+-/
+
+import category_theory.limits.shapes
```

Comment:
```suggestion
import category_theory.limits.shapes.equalizers
import category_theory.limits.shapes.binary_products
import category_theory.limits.shapes.pullbacks
```
I prefer to import only exactly what is needed. Saves headaches later!

Reply:
Edited, thanks!